101-vm-from-user-image/azuredeploy-parameters.json

```diff
@@ -1,44 +1,26 @@
 ﻿{
     "adminUsername": {
-        "value": "cloudguy"
+        "value": "username"
     },
     "adminPassword": {
-        "value": "abc_123!"
+        "value": "password!"
     },
     "userImageStorageAccountName": {
-        "value" : "clodfed13"
+        "value" : "myStorageAccount"
     },
     "userImageStorageContainerName" : {
-        "value" : "vhds"
+        "value" : "myContainerName"
     },
     "userImageVhdName" : {
         "value" : "sample.vhd"
     },
     "dnsNameForPublicIP": {
-        "value": "clodfed13"
+        "value": "uniqueDNS"
     },
     "osType" : {
         "value" : "linux"
     },
-    "subscriptionId": {
-        "value": "87a9cd1c-aa24-446d-b7e0-d73955447c69"
-    },
-    "location": {
```

Comment:
I know I suggested this, but does it work in the Deploy button without location?